src/attestation.nim

```diff
@@ -217,9 +217,13 @@ proc loadFromSecretManager*(prkey: string, apikey: string): bool =
 
   return true
 
-proc getCosignLocation*(): string =
+proc getCosignLocation*(downloadCosign = false): string =
   once:
-    cosignLoc = unpack[string](runCallback(cosignLoader, @[]).get())
+    var args = @[pack(false)]
+    if downloadCosign == true:
+      args = @[pack(true)]
```

Comment:
Just a nit here, I'd probably just do:
```
let args = @[pack(downloadCosign)]
```

Reply:
I think it makes it much clearly what's going on, in its simplicity.

Reply:
Yup agreed, makes sense